images/ubuntu/toolsets/toolset-2004.json

```diff
@@ -45,11 +45,11 @@
             "arch": "x64",
             "platform" : "linux",
             "versions": [
-                "1.20.*",
                 "1.21.*",
-                "1.22.*"
+                "1.22.*",
+                "1.23.*"
             ],
-            "default": "1.21.*"
```

Comment:
I can see announcement for changing default version, however target version 1.22 and not 1.23 - #10500